pandas/core/generic.py

```diff
@@ -3349,8 +3349,8 @@ def to_csv(
             sequence should be given if the object uses MultiIndex. If
             False do not print fields for index names. Use index_label=False
             for easier importing in R.
-        mode : str
-            Python write mode, default 'w'.
+        mode : str, default 'w'
+        Python write mode. The available write modes are the same as :py:func:`open`
```

Comment:
Looks good! This will just need indenting similar to how it was before.

Reply:
Alright, will do it